code/controllers/subsystem/processing/resinshaping.dm

```diff
@@ -1,20 +1,21 @@
 SUBSYSTEM_DEF(resinshaping)
 	name = "Resin Shaping"
 	flags = SS_NO_FIRE
-	/// Counter for quickbuilds, as long as this is above 0 building is instant.
-	var/quickbuilds = 0
+	/// Counter for quickbuild points, as long as this is above 0 building is instant.
+	var/list/quickbuild_points_by_hive = list()
 	/// Whether or not quickbuild is enabled. Set to FALSE when the game starts.
 	var/active = TRUE
 
 /datum/controller/subsystem/resinshaping/stat_entry()
-	..("QUICKBUILDS=[quickbuilds]")
+	..("QUICKBUILD POINTS (NORMAL HIVE)=[quickbuild_points_by_hive[XENO_HIVE_NORMAL]]")
```

Comment:
This is only for the admin MC tab, right?

Reply:
Yeah, don't think it's worth displaying the points for all hives